Source/Scene/Model.js

```diff
@@ -3342,6 +3354,22 @@ define([
         };
     }
 
+    function triangleCountFromPrimitiveIndices(primitive, indicesCount) {
+        switch (primitive.mode) {
+            case PrimitiveType.POINTS:
```

Comment:
All these cases that return 0 here can be removed; they are covered by the return 0 below.